formatters/source_file.go

```diff
@@ -24,11 +24,19 @@ type SourceFile struct {
 }
 
 func (a SourceFile) Merge(b SourceFile) (SourceFile, error) {
-	if len(a.Coverage) != len(b.Coverage) {
-		return a, errors.Errorf("coverage length mismatch for %s", a.Name)
+  if a.BlobID != b.BlobID {
+		return a, errors.Errorf("Failed to merge coverage for source file %s. BlobID mismatch", a.Name)
```

Comment:
It's a little odd to treat to the first part as a proper sentence with a period but the end of the message is a sentence fragment without punctuation. Can we use : instead of . so the message is more similar in grammatical structure to other messages?